src/main/scala/io/sdkman/changelogs/QuarkusMigrations.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "024")
+class QuarkusMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_quarkus_candidate",
+    author = "maxandersen"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "quarkus",
+      name = "Quarkus",
+      description =
+        "Quarkus is a Kubernetes Native Java framework tailored for OpenJDK HotSpot and GraalVM, crafted from best-of-breed Java libraries and standards.",
```

Comment:
Is this just the CLI component? If so, can we make that explicit?

Reply:
I was looking at micronaut for uniform messaging and it does not say only cli even though it seem to be just a cli.

I can change it but what todo with micronaut one ?

Reply:
It's not a biggie I guess. I noticed that Spring Boot also doesn't mention anything about CLI.

Reply:
okey :) so we are good to go on this ?